database/data/dataGeneration/genSongsCASSANDRA.js

```diff
@@ -0,0 +1,50 @@
+const faker = require('faker');
+const fs = require('fs');
+
+const generateUserList = () => {
+    const users = [];
+    for (let i = 0; i < 1000; i += 1) {
+      users.push(faker.internet.userName());
+    }
+    return users;
+  };
+
+const users = generateUserList();
+
+const writeSongs = fs.createWriteStream('/Users/jonathanolson/HackReactor/SDC/playbar/database/data/songsCASSANDRAREMAINDER.csv');
+writeSongs.write('songId,likeId,album,artist,likeCount,likeUserName,songDataURL,songName,thumbnailURL\n', 'utf8');
+
+
+const writeRemainingSongs = (writer, encoding, callback) => {
+    let i = 10000000;
+    function write() {
+        let ok = true;
+        do {
+        i -= 1;
+        const userName =
+        users[
+        faker.random.number({
+            min: 0,
+            max: 999,
+        })
+        ];
+        const data = `${i},${-1},${faker.random.word()},${userName},${faker.random.number()},${'likeUserName_example'},${faker.image.imageUrl()},${faker.random.word().slice()},${faker.image.imageUrl()}\n`;
+        if (i === 8385062) {
```

Comment:
what does this number mean? `8385062`

Reply:
My computer ran out of memory during seeding and this was the first index that was not seeded, so it was my starting point for data generation to seed the remaining values.